src/utils/states.js

```diff
@@ -50,21 +50,26 @@ export const STATE_NAMES = [
   'All',
   'Success',
   'Failed',
+  'Finished',
   'Scheduled',
+  'ScheduledAlt',
```

Comment:
Where does this state come from?

Reply:
In the `states.js` file then in the `STATE_COLORS`. Should this be removed? I was also referring to this [diagram](https://docs.prefect.io/api/latest/engine/state.html#state-2)

Reply:
this is there so we can reference the color for scheduled flows (its an alternating yellow stripe). If this is a bad place for it, we can move it to a var in vuetify.js instead

Reply:
Hm yeah - this isn't the correct place for it, since it'll be included in state lists (and `ScheduledAlt` isn't a valid state)